packages/nextjs/components/ExtensionCard.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from "react";
+import { usePlausible } from "next-plausible";
+import CopyToClipboard from "react-copy-to-clipboard";
+import { CheckCircleIcon, DocumentDuplicateIcon } from "@heroicons/react/24/outline";
+import { Address } from "~~/components/scaffold-eth";
+
+type Extension = {
+  name: string;
+  description: string;
+  github: string;
+  installCommand: string;
+  builder: string;
+  coBuilders: string[];
+  youtube?: string;
+};
+
+export const ExtensionCard = ({ extension, isCurated }: { extension: Extension; isCurated: boolean }) => {
+  const [commandCopied, setCommandCopied] = useState(false);
+  const plausible = usePlausible();
+
+  const handleInstallClick = () => {
+    setCommandCopied(true);
+    setTimeout(() => {
+      setCommandCopied(false);
+    }, 800);
+
+    const githubRepo = isCurated
+      ? `Curated/${extension.name}`
+      : extension.github.split("github.com/")[1] || extension.github;
+
+    // Track the click event with GitHub repo as a prop
+    plausible("extensionCopyClick", { props: { id: githubRepo } });
+  };
+
+  return (
+    <div className="card bg-base-100 shadow-xl mb-8 flex flex-col mx-1">
+      <div className="card-body flex-grow">
+        <h2 className="card-title">{extension.name}</h2>
+        <div className="flex justify-between items-start mb-2">
+          <div className="flex items-center gap-2">
+            {extension.github && (
+              <a href={extension.github} className="inline-block" target="_blank" rel="noopener noreferrer">
+                {
+                  // eslint-disable-next-line
+                  <img alt="github icon" className="w-6 h-6" src="/icon-github.svg" />
```

Comment:
Any reason you are not using `Image` from `next/image`. There is an example on the index page with the GitHub icon,

Reply:
No reason, bad habits! 😅 Fixed in https://github.com/scaffold-eth/scaffoldeth.io/pull/35/commits/0ed70c8955da06e2769d7018e2994f5e50e4acb6